lib/Service/FileService.php

```diff
@@ -60,57 +78,74 @@
 class FileService
 {
     /**
-     * Root folder name for all OpenRegister files
+     * Root folder name for all OpenRegister files.
+     *
+     * @var        string
+     * @readonly
+     * @psalm-readonly
      */
-    const ROOT_FOLDER = 'Open Registers';
+    private const ROOT_FOLDER = 'Open Registers';
```

Comment:
A small bit of nitpicking, but according to classic coding conventions constants should actually be either public or protected. If there's good reason to keep it private it is good to motivate that.